### ISSUE TITLE
The datashader does not work on python 3.8

### ISSUE THREAD

**Issue by scarabeusiv** [holoviz/datashader#809]
At minimal the collections import syntax changed and as such it needs to be tweaked:

```
[   99s] datashader/transfer_functions.py:3
[   99s]   /home/abuild/rpmbuild/BUILD/datashader-0.8.0/datashader/transfer_functions.py:3: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
[   99s]     from collections import Iterator, OrderedDict
```

**Comment by jbednar** [comment]
Thanks for the heads up.  Would you be able to put together a PR with the fixes?  It seems straightforward ("from collections.abc import ..."), and it sounds like you are in a position to test it out...

**Comment by jbednar** [comment]
From https://docs.python.org/3/whatsnew/3.8.html#api-and-feature-removals , it doesn't look like we actually need to make this change for 3.8, as importing from `collections` only was postponed to 3.9.  We should still do it, but I think we should do it as part of testing Datashader on 3.8 more generally. To that end, `conda create -n py38 python=3.8 datashader` doesn't seem to succeed right now, as does `conda create -n py38 python=3.8 ; conda activate py38 ; conda install --only-deps datashader`, suggesting that some dependency of datashader is not yet available for 3.8; I'll have to investigate.

**Comment by philippjfr** [comment]
This was fixed in https://github.com/holoviz/datashader/pull/859 and numba incompatibilities with Python 3.8 were solved a while back.
